@xen-orchestra/lite/src/components/pool/dashboard/cpuUsage/HostsCpuUsage.vue

```diff
@@ -50,4 +53,8 @@ const statFetched: ComputedRef<boolean> = computed(() =>
     ? true
     : stats.value.length > 0 && stats.value.length === data.value.length
 );
+
+const statHasError: ComputedRef<boolean> = computed(() =>
+  stats.value.every((stat) => stat.stats === null)
+);
```

Comment:
I think you can directly use `hostStore.hasError` since `stat.stats === null` only if `hostStore.hasError`. This will avoid going through all the stats.